src/components/messagebox/ChattingRoom/index.tsx

```diff
@@ -1,114 +1,150 @@
-import { Image, IconButton, Input } from '@offer-ui/react'
+import { Image, IconButton, Input, useMedia } from '@offer-ui/react'
+import { useEffect, useRef, useState } from 'react'
 import { Styled } from './styled'
 import type { ChattingRoomProps } from './types'
 import { Chatting } from '../Chatting'
+import type { ChattingProps } from '../Chatting/types'
+import { Dialog } from '@components/common'
 import { toLocaleCurrency } from '@utils/format'
+import {
+  useCreateMessageMutation,
+  useDeleteMessageRoomMutation,
+  useGetMessageQuery
+} from '@apis'
+import { useAuth, useModal } from '@hooks'
 
-export const ChattingRoom = ({ id, onClose }: ChattingRoomProps) => {
-  // TODO: ChattingRoom Id를 통해서 컴포넌트 내부에서 패치하도록
+// TODO: messageRoom 정보조회 api 붙이고 제거
+const POST_MOCK = {
+  offerPrice: 10000,
+  post: {
+    title: '팔아요 !',
+    thumbnailImageUrl: '',
+    price: 12000
+  }
+}
+
+export const ChattingRoom = ({ roomId, onClose }: ChattingRoomProps) => {
+  const getMessageQuery = useGetMessageQuery({
+    msgRoomId: roomId,
+    page: 0
+  })
+  const createMessageMutation = useCreateMessageMutation(roomId)
+  const deleteMessageRoomMutation = useDeleteMessageRoomMutation(roomId)
+  const chattingBoxRef = useRef<HTMLDivElement | null>(null)
+  const inputRef = useRef<HTMLInputElement | null>(null)
+  const { desktop, tablet, mobile } = useMedia()
+  const [messages, setMessages] = useState<ChattingProps['messages']>([])
+  const { isOpen, openModal, closeModal } = useModal()
+  const { user } = useAuth()
+  const senderInfo = {
+    id: user.id,
+    nickname: user.nickname,
+    imageUrl: user.profileImageUrl
+  }
+
+  const scrollToBottom = () => {
+    if (!chattingBoxRef.current) {
+      return
+    }
+
+    if (chattingBoxRef.current.scrollHeight > 0) {
+      chattingBoxRef.current.scrollTop = chattingBoxRef.current.scrollHeight
+    }
+  }
 
   const handleCloseRoom = () => {
-    onClose?.(id)
+    onClose?.(roomId)
+  }
+
+  const handleDeleteRoom = async () => {
+    await deleteMessageRoomMutation.mutateAsync()
+
+    handleCloseRoom()
   }
 
+  const refetchMessage = async () => {
+    await getMessageQuery.refetch()
+  }
+
+  const handleSubmitMessage = async (message: string) => {
```

Comment:
```suggestion
  const handleSubmitMessage = async (content: string) => {
```
아예 content로 받아서 축약해도 될 것 같습니다!

Reply:
[변경 완료 ! ](https://github.com/price-offer/offer-fe/pull/196/commits/c334aa356693e396051ebe90551f92c71661cf30#diff-2e1f52de9ff48c1e322340a310c75b772aa52c734108dbf9888ba96a88db7ce3R68)